### PR TITLE
CLOUDP-193003: Add command “atlas backups compliancepolicy policies describe”

### PR DESCRIPTION
**Pull request by VincentPikand**
## Proposed changes

Added the command `atlas backups compliancepolicy policies describe`, which lists the individual policy items that a backup complinace policy has. Functionally identical to `atlas backups compliancepolicy describe`.

### command output
```
$ atlas backups compliancepolicy policies describe
POLICIES
ID                         FREQUENCY INTERVAL   FREQUENCY TYPE   RETENTION
64b16beffcaed70cfbd5f520   -                    daily            2 months
64ba452852594888bd5b7070   1                    hourly           1 days
64c795d6401a8ccaea00bf85   -                    ondemand         1 days
```

_Jira ticket:_ CLOUDP-193003

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/backup/compliancepolicy/compliancepolicy.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ package compliancepolicy
 
 import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	policies "github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/backup/compliancepolicy/policies"
```
Comment (gssbzn) [inline internal/cli/atlas/backup/compliancepolicy/compliancepolicy.go:19]:
```suggestion
	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/backup/compliancepolicy/policies"
```
you shouldn't need an alias here sine is the same as the pkg name

File: internal/cli/atlas/backup/compliancepolicy/copyprotection.go [hunk 2]
```diff
@@ -42,7 +42,9 @@ const (
 	active  = "ACTIVE"
 )
 
-var copyProtectionTemplate = `Copy protection has been set to: {{.CopyProtectionEnabled}}`
+var copyProtectionTemplate = `
+Copy protection has been set to: {{.CopyProtectionEnabled}}
```
Comment (gssbzn) [inline internal/cli/atlas/backup/compliancepolicy/copyprotection.go:46]: any reason to add a new line at the beginning of the message?

**Comment by gssbzn** [comment]
can you please include in the PR description the output of running the command?

**Review by gssbzn — APPROVED** [review]

**Review by tibulca — APPROVED** [review]
LGTM

**Pull request merged**